app/style.css

```diff
@@ -0,0 +1,285 @@
+.hide{
```

Comment:
Try to use comments to get your CSS organised. For instance, you could create sections with comments for: `common-classes`, `header`, `footer` and so on